selfdrive/car/gm/values.py

```diff
@@ -85,6 +86,7 @@ class GMCarInfo(CarInfo):
   CAR.ACADIA: GMCarInfo("GMC Acadia 2018", video_link="https://www.youtube.com/watch?v=0ZN6DdsBUZo"),
   CAR.BUICK_REGAL: GMCarInfo("Buick Regal Essence 2018"),
   CAR.ESCALADE_ESV: GMCarInfo("Cadillac Escalade ESV 2016", "Adaptive Cruise Control (ACC) & LKAS"),
+  CAR.BOLT_EV: GMCarInfo("Chevrolet Bolt EV 2022-23", "Adaptive Cruise Control (ACC)", footnotes=[], harness=Harness.gm),
```

Comment:
Brochure mentions ACC "requires available Infotainment Package," however the Infotainment Package does ***not*** come with ACC, you need to equip it as its own feature on top of the Infotainment Package.